src/parenthetics.py

```diff
@@ -0,0 +1,26 @@
+"""Test if string of parentheses is open, balanced, or broken with the use of a data structure."""
+
+from stack import Stack
+
+
+def proper_parenthetics(string):
+    """Function tests if string is open, balanced, or broken."""
+    new_stack = Stack()
```

Comment:
So why did you need the Stack again?

Reply:
Probably corpse code. I was trying to use a data structure, and that obviously didn't work out...